react/features/conference/components/Conference.native.js

```diff
@@ -360,6 +363,9 @@ function _mapDispatchToProps(dispatch) {
          * Dispatches actions to create the desired local tracks and for
          * connecting to the conference.
          *
+         * FIXME: Do these have to be here? Shouldn't this be in a middleware
```

Comment:
Please remove this. If we need to look into it open an issue, adding this won't magically fix it :-P

Reply:
Well, I was hoping that someone will see it when he's working around this area and decides to refactor it 🌞